scripts/ci

```diff
@@ -3,10 +3,10 @@ set -e
 
 cd $(dirname $0)
 
-./build
-./test
```

Comment:
Arguably static analysis and other forms of validation should come before building and running tests.